javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+var $lightbulb = document.querySelector('.bulb-on');
+var $darklight = document.querySelector('.light-on');
+var counter = 0;
+
+function turnLightBulb() {
+  counter++;
+  if (counter % 2 === 0) {
```

Comment:
A count of the clicks will work here, but you are keeping track of a count and just determining if that count is even or odd, one of two states. A boolean would be a better way to keep track of whether something is one of two different states. You should make a variable to keep track of if the light is currently on, with a boolean value.

Reply:
<img width="971" alt="Screen Shot 2022-07-15 at 4 54 50 PM" src="https://user-images.githubusercontent.com/97363006/179325659-8ba74b24-4d4a-4a20-a8dd-7fb840d00f09.png">

Reply:
<img width="1030" alt="Screen Shot 2022-07-25 at 3 45 36 PM" src="https://user-images.githubusercontent.com/97363006/180886828-e1384caf-a3e7-4239-b0a7-74db53d668ad.png">